### PR TITLE
Assignment interface

### PR DESCRIPTION
**Pull request by bjones1**
Fixes as requested in #957.

### CODE REVIEW

File: static/js/admin.js [hunk 1]
```diff
@@ -1295,117 +1301,6 @@ function remove_question(question_name) {
...
-        });
-    };
-
-    var hiddenwrite = document.getElementById('hiddenwrite');
-    hiddenwrite.style.visibility = 'visible';
-}
-
-
-// Called when the "Done" button of the "Write" dialog is clicked.
-function create_question(formdata) {
```
Comment (bnmnetp) [inline static/js/admin.js]: This seems a bit overzealous.  We still want people to be able to write their own questions, preview them, and have them added to the DB.

File: views/admin/assignments.html [hunk 2]
```diff
@@ -33,15 +33,31 @@
...
-<div class="col-md-6">
-  <h4 style="text-align: center;">Assignment</h4>
-  <select class="form-control" id="assignlist" onchange="assignmentInfo();">
-    {{ for assignmentid in assignments: }}
-    <option value="{{=assignmentid}}">{{=assignments[assignmentid]}}</option>
-    {{ pass }}
-  </select>
-  <button class="btn" data-target="#addAssignment" data-toggle="modal" style="margin-top: 12px;">+</button>
-  <button class="btn" onclick="remove_assignment();" style="margin-top: 12px;">-</button>
+<div class="row">
```
Comment (bnmnetp) [inline views/admin/assignments.html]: I'll study these a bit more, but as I look at the interface, it seems that the ability to write your own questions for an assignment is completely gone.  Why?

**Comment by bnmnetp** [comment]
OK, maybe there was confusion about what I meant when I said Write Assignment.  I meant that there is no need for the button in this screenshot to exist anymore.  But we definitely still want instructors to be able to write and add questions to the database.

screen shot 2017-07-25 at 12 11 14 pm

**Comment by bjones1** [comment]
Oops! So, am I correct that I should:

- [ ] Revert the commit that removed the "Write" button.
- [ ] Remove the "Write Assignment" button and all the code behind it.

**Comment by bnmnetp** [comment]
Yes, that is correct.

In the new world (which you should like!) The assignment page is generated dyamically on the server side.  Removing the Write Asssignment button will remove the code that updates the assignments.rst file and creates a new assignment in the assignments folder.  --- YAY! NO NEED to REBUILD your course just because you made an assignment.   Once you've restored the write button, questions written and added to assignments will work too!


I'll have to remember this, because this is a great example of why you should do good feature based commits !

**Comment by bjones1** [comment]
OK, this part should be fixed.

**Pull request closed without merging**